### PR TITLE
feature/ads-643 Monorepo

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+/**/node_modules
 /.pnp
 .pnp.js
 
 # testing
 /coverage
 
 # production
-/build
+/docs//build
```
Comment (kaseyllabelle) [inline .gitignore:12]: // intentional?

File: .prettierignore [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Ignore artifacts:
-build
+docs/build
 coverage
-node_modules
-cypress
-src/docs/shared/types/contentful.ts
-schema.graphql
+**/node_modules
```
Comment (kaseyllabelle) [inline .prettierignore:4]: some differences in / vs ** in this and gitignore. intentional?

File: docs/src/docs/shared/versions.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const tokens = '3.1.1';
```
Comment (kaseyllabelle) [inline docs/src/docs/shared/versions.ts:1]: this is for the version in the tokens page, right?

File: docs/src/docs/App.tsx
Comment (kaseyllabelle) [inline docs/src/docs/App.tsx:1]: it feels a little weird to have `docs/src/docs` is this a temporary thing?

Comment (APJohns) [reply docs/src/docs/App.tsx:1]: Yes, it will just docs/src once I pull out library from docs

File: .prettierignore (hunk 2 above)
Comment (APJohns) [reply .prettierignore]: ** is to ignore all the node_modules folders in the monorepo

Comment (kaseyllabelle) [reply .prettierignore]: i meant that we have /docs/build vs docs/build and /**/node vs **/node

**Review by kaseyllabelle — APPROVED** [review]

**Review by kaseyllabelle — APPROVED** [review]

**Pull request merged**